src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -40,10 +45,12 @@
             </div>
           </div>
         </ng-container>
-        &nbsp;
         <div class="field">
           <div *ngIf="!f.valid" class="error">
-            <sup>*</sup> Field can't left blank.
+            <sup>*</sup> Field can't be left blank.
```

Comment:
We need to confirm this, config item allows itself to be blank. 

*And DELETE call reset to its default value